packages/Webkul/Shop/src/Resources/lang/vi/app.php

```diff
@@ -0,0 +1,935 @@
+                    ],
+                ],
+
+                'custom-filters' => [
+                    'clear-all' => 'Xóa tất cả',
+                    'title'     => 'Bộ lọc tùy chỉnh',
+                ],
+
+                'date-options' => [
+                    'last-month'        => 'Tháng trước',
+                    'last-six-months'   => '6 tháng trước',
+                    'last-three-months' => '3 tháng trước',
+                    'this-month'        => 'Tháng này',
+                    'this-week'         => 'Tuần này',
+                    'this-year'         => 'Năm nay',
+                    'today'             => 'Hôm nay',
+                    'yesterday'         => 'Hôm qua',
+                ],
+            ],
+
+            'table' => [
+                'actions'              => 'Hành động',
+                'next-page'            => 'Trang kế',
+                'no-records-available' => 'Không có dữ liệu.',
+                'page-navigation'      => 'Điều hướng trang',
+                'page-number'          => 'Số trang',
+                'previous-page'        => 'Trang trước',
+            ],
+
+        ],
+
+        'modal' => [
+            'default-content' => 'Nội dung mặc định',
+            'default-header'  => 'Tiêu đề mặc định',
+
+            'confirm' => [
+                'agree-btn'    => 'Đồng ý',
+                'disagree-btn' => 'Không đồng ý',
+                'message'      => 'Bạn có chắc chắn muốn thực hiện hành động này không?',
+                'title'        => 'Bạn có chắc chắn không?',
+            ],
+        ],
+
+        'products' => [
+            'card' => [
+                'add-to-cart'            => 'Thêm vào giỏ hàng',
+                'add-to-compare'         => 'Thêm vào so sánh',
+                'add-to-compare-success' => 'Sản phẩm được thêm vào danh sách so sánh thành công.',
+                'add-to-wishlist'        => 'Thêm vào danh sách mong muốn',
+                'already-in-compare'     => 'Sản phẩm đã được thêm vào danh sách so sánh.',
+                'new'                    => 'Mới',
+                'review-description'     => 'Hãy là người đầu tiên đánh giá sản phẩm này',
+                'sale'                   => 'Giảm giá',
+            ],
+
+            'carousel' => [
+                'next'     => 'Tiếp theo',
+                'previous' => 'Trước đó',
+                'view-all' => 'Xem tất cả',
+            ],
+        ],
+
+        'range-slider' => [
+            'max-range' => 'Phạm vi tối đa',
+            'min-range' => 'Phạm vi tối thiểu',
+            'range'     => 'Phạm vi:',
+        ],
+
+
+        'carousel' => [
+            'image-slide' => 'Ảnh Slide',
+            'next'        => 'Tiếp theo',
+            'previous'    => 'Trước đó',
+        ],
+
+        'quantity-changer' => [
+            'decrease-quantity' => 'Giảm Số lượng',
+            'increase-quantity' => 'Tăng Số lượng',
+        ],
+
+        'products' => [
```

Comment:
All arrays and keys should be adjusted to ensure consistency. This array is placed under 'component' it should be outside of component please match all the arrays and keys and remove extra spaces take the reference from 'en' (English) locale.

Reply:
Update 
![image](https://github.com/bagisto/bagisto/assets/10725732/81237f72-f4a1-47ef-b7fe-1476e729b32b)